pipelines/branch.yml

```diff
@@ -335,7 +335,9 @@ jobs:
     - get: unit-image
       passed: [k8s-smoke]
       trigger: true
-    - get: charts
+    - get: helm-charts
+      passed: [k8s-smoke]
```

Comment:
Hey,

is this new passed constraint necessary? If I got it right, this one will make setting the pipeline fail, as `k8s-smoke` does not `get` `helm-charts` 🤔  

If it is, then we'd need to add it there (have a `get`), or, make it `passed` on something that uses it.

thx!